variables.tf

```diff
@@ -28,4 +28,9 @@ variable "app_type" {
 
 variable "aws_profile" {
   type = string
+}
+
+variable "ttl" {
+  type = number
+  default = 300
```

Comment:
Why we need "ttl" as variable?

Reply:
ttl variable is the ttl of the dns record , this is for the issue when we do destroy before the dns cache is changed, so I added a sleep by the ttl of the record:
https://toluna.atlassian.net/jira/software/projects/PLM/boards/137?assignee=557058%3Aa65bf53b-7bf0-4e26-9ec8-f60696fc173a&selectedIssue=PLM-513